lib/src/tar/export.rs

```diff
@@ -541,8 +541,12 @@ pub(crate) fn export_chunk<W: std::io::Write>(
         let (objpath, h) = writer.append_content(checksum.borrow())?;
         for path in paths.iter() {
             let path = path.strip_prefix("/").unwrap_or(path);
+            let path = match path.strip_prefix("usr/etc") {
```

Comment:
This change looks good on its own, but I guess `export_final_chunk()` may need the same kind of path tweaking?

Reply:
Ooh, you're absolutely right.  I think in practice it wasn't a problem, but yes.